components/src/core/Drawer/DrawerNavItem/DrawerNavItem.tsx

```diff
@@ -22,6 +22,7 @@ import drawerNavItemClasses, {
 } from './DrawerNavItemClasses';
 import Box, { BoxProps } from '@mui/material/Box';
 import { unstable_composeClasses as composeClasses } from '@mui/base';
+import type {} from '@mui/material/themeCssVarsAugmentation';
```

Comment:
Same comment as before...we should ideally only have to have these type augmentation once in the root index file.